app/src/main/kotlin/com/aliucord/manager/ui/component/home/InfoCard.kt

```diff
@@ -0,0 +1,116 @@
+package com.aliucord.manager.ui.component.plugins
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.*
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.*
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.aliucord.manager.R
+
+@Composable
+fun InfoCard(
+    packageName: String,
+    supportedVersion: String,
+    currentVersion: String,
+    onDownloadClick: () -> Unit,
+    onUninstallClick: () -> Unit,
+    onLaunchClick: () -> Unit
+) {
+    ElevatedCard {
+        Column(
+            modifier = Modifier
+                .padding(20.dp)
+                .fillMaxWidth(),
+            verticalArrangement = Arrangement.spacedBy(8.dp)
+        ) {
+            Text(
+                text = "Aliucord${packageName.let { if (it != "com.aliucord") " ($it)" else "" }}",
+                style = MaterialTheme.typography.titleMedium.copy(
+                    fontSize = 20.sp
+                ),
```

Comment:
The "Aliucord' header is way too small for the core label of Manager.
(also font weird)
Can you try reverting to how it was before